### PR TITLE
chore: refactoring react hooks

### PR DESCRIPTION
**Pull request by zkdan**
Changing React.useEffect to useEffect, mostly.

There are too many commits because I was working on two things at once - the latest ones are the relevant ones. 
Created a type for collections (named it ICollections even though it's a type not an interface - was merely trying to get the linter to stop yelling).

### CODE REVIEW

**Review by Yukthiw — APPROVED** [review]
LGTM

**Review by alexwaeseperlman — APPROVED** [review]

File: Eplant/util/Storage/index.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default class KeyValDB<K extends string, V> {
   }
 
   async clear() {
-    console.log(await this.keys())
+    // console.log(await this.keys())
```
Comment (mwkyuen) [inline Eplant/util/Storage/index.tsx:67]: Perhaps we can remove this? I'm not sure why we are keeping it as a comment

File: Eplant/util/ErrorBoundary/index.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ export default class ErrorBoundary extends React.Component<
     this.state = { hasError: false }
   }
   componentDidCatch(error: Error, info: ErrorInfo) {
-    console.log(error, info)
+    // console.log(error, info)
```
Comment (mwkyuen) [inline Eplant/util/ErrorBoundary/index.tsx:11]: Perhaps we can remove this? I'm not sure why we are keeping it as a comment

File: Eplant/state/index.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ export const pageLoad = (() => {
     },
     done() {
       finished++
-      console.log(finished / waiting)
+      // console.log(finished / waiting)
```
Comment (mwkyuen) [inline Eplant/state/index.tsx:28]: Perhaps we can remove this? I'm not sure why we are keeping it as a comment

File: Eplant/main.tsx [hunk 4]
```diff
@@ -71,11 +71,11 @@ const registerServiceWorker = async () => {
     try {
       const registration = await navigator.serviceWorker.register(import.meta.env.BASE_URL + '/sw.js')
       if (registration.installing) {
-        console.log('Service worker installing')
+        // console.log('Service worker installing')
```
Comment (mwkyuen) [inline Eplant/main.tsx:74]: Here too. Perhaps we can remove this? I'm not sure why we are keeping it as a comment

**Review by mwkyuen — APPROVED** [review]
Minor comment, but LGTM!

**Pull request closed without merging**